orchagent/orchdaemon.cpp

```diff
@@ -252,6 +252,7 @@ bool OrchDaemon::init()
     gDirectory.set(dash_route_orch);
 
     vector<string> dash_acl_tables = {
+        "DASH_PREFIX_TAG_TABLE",
```

Comment:
Why don't we define a macro just like other tables in swss common?

Reply:
I even created a PR for that in swss-common :)
Just forgot to replace the string